src/components/Web3Handlers/Web3SuccessHandler.tsx

```diff
@@ -19,5 +20,17 @@ export const Web3SuccessHandler: React.FC<Props> = ({ hash }) => (
         {utils.formatting.shortenAddress(hash)}
       </a>
     </span>
+    <span className="hidden lg:block">
+      {`${message}: `}
+      <br />
+      <a
+        className="underline text-blue-600 hover:text-blue-800 visited:text-purple-600"
+        href={`${process.env.VITE_SCANNER_URL}/tx/${hash}`}
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        {hash}
```

Comment:
to shorten the hash you can give the `<a>` a maximum width, and use the `truncate` class name to achieve the same effect as `shortenAddress` in this case

Reply:
We don't want to shorten the hash here, only on the small screen. Good to know that is an option but for the sake of consistency I want to stick with shortenAddress as not all hashes may not be links. We also have a unit test for the shortenAddress method

Reply:
in that case i would do the showing / hiding of elements just in the `<a>` tag - but up to you 🤷